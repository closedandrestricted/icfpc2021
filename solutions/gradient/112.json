{"vertices":[[43,38],[133,59],[61,66],[128,84],[128,52],[136,29],[66,67],[100,33],[119,44],[39,91],[107,67],[100,66],[102,98],[115,182],[72,99],[116,151],[127,120],[105,100],[95,131],[152,67],[93,130],[79,174],[137,96],[122,152],[95,133],[125,120],[181,55],[64,116],[95,139],[167,81],[148,136],[171,95],[155,110],[126,118],[92,134],[94,126],[186,78],[62,149],[167,153],[176,125],[189,103],[189,175],[193,147]]}