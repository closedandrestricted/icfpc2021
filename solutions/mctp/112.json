{"vertices":[[114,132],[28,159],[80,126],[46,176],[77,171],[15,134],[80,125],[44,181],[28,152],[84,160],[54,149],[54,149],[85,154],[7,91],[52,150],[39,95],[57,120],[40,181],[92,122],[48,95],[52,151],[32,66],[80,104],[73,150],[91,121],[60,120],[70,74],[21,141],[64,87],[50,94],[88,123],[61,88],[81,97],[57,120],[51,153],[88,115],[61,66],[59,121],[69,141],[62,114],[71,89],[63,112],[41,96]]}