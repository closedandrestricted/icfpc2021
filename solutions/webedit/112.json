{"vertices": [[39, 96], [79, 149], [73, 95], [62, 131], [78, 105], [57, 169], [62, 69], [62, 76], [77, 160], [97, 71], [89, 137], [95, 79], [82, 107], [85, 157], [106, 38], [94, 127], [96, 170], [126, 64], [97, 136], [104, 67], [111, 92], [114, 178], [116, 97], [125, 154], [96, 136], [126, 121], [133, 56], [140, 77], [141, 152], [145, 82], [150, 137], [151, 90], [155, 111], [94, 171], [157, 105], [91, 139], [169, 77], [170, 135], [175, 142], [180, 114], [191, 90], [144, 140], [153, 113]]}