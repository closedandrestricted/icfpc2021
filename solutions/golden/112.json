{"vertices":[[42,34],[129,60],[60,64],[124,85],[127,54],[137,31],[63,64],[101,32],[119,44],[36,88],[105,66],[98,65],[101,97],[114,181],[71,95],[116,150],[125,119],[102,100],[94,130],[165,82],[92,129],[78,173],[148,110],[121,151],[93,132],[124,119],[180,56],[59,130],[94,138],[168,83],[148,136],[174,102],[154,111],[124,116],[91,133],[93,126],[195,93],[61,148],[167,154],[175,126],[171,100],[191,173],[195,145]]}